examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>May be don't show all the metrics but only mean + good case + bad case just to illustrate the words </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
<p>I think it is better to show them all, as it will help to emphasise the differences and similarities between methods.</p>